### PR TITLE
[Launcher]: Add RetryPolicy Doc

### PR DESCRIPTION
**Pull request by yqwang-ms**

### CODE REVIEW

**Comment by fanyangCS** [comment]
looks like the focus of this PR is retryPolicy. not CompletionPolicy.

File: frameworklauncher/doc/USERMANUAL.md [hunk 1]
```diff
@@ -633,24 +639,103 @@ Notes:
...
 ApplicationCompletionPolicy can be configured for each TaskRole to control:
-1. The conditions to complete the Application.
-2. The ExitStatus of the completed Application.
+1. The conditions to complete the [Application](#Concepts_YARN).
+2. The ExitStatus of the completed [Application](#Concepts_YARN).
 
 ### <a name="ApplicationCompletionPolicy_Usage">Usage</a>
 For details, please check: [TaskRoleApplicationCompletionPolicyDescriptor](../src/main/java/com/microsoft/frameworklauncher/common/model/TaskRoleApplicationCompletionPolicyDescriptor.java).
 
 ### <a name="ApplicationCompletionPolicy_Examples">Examples</a>
```
Comment (fanyangCS) [inline frameworklauncher/doc/USERMANUAL.md]: >Examples [](start = 51, length = 8)

it is weird you firstly introduce "MinFailedTaskCount" and "MinSucceededTaskCount" in an example. do you have anywhere else to explain the concept formally?

**Comment by yqwang-ms** [comment]
Yes, CompletionPolicy is already checked in, but anyway you can review it :)

---
In reply to: [402612758](https://github.com/Microsoft/pai/pull/812#issuecomment-402612758) [](ancestors = 402612758)

File: frameworklauncher/doc/USERMANUAL.md (hunk 1 above)
Comment (yqwang-ms) [reply frameworklauncher/doc/USERMANUAL.md]: They are in previous section [Usage] with hyperlink.

View it in the web is more friendly:
[https://github.com/Microsoft/pai/blob/yqwang/launcher-dev/frameworklauncher/doc/USERMANUAL.md#ApplicationCompletionPolicy](https://github.com/Microsoft/pai/blob/yqwang/launcher-dev/frameworklauncher/doc/USERMANUAL.md#ApplicationCompletionPolicy)


### <a name="ApplicationCompletionPolicy_Usage">Usage</a>
For details, please check: [TaskRoleApplicationCompletionPolicyDescriptor](../src/main/java/com/microsoft/frameworklauncher/common/model/TaskRoleApplicationCompletionPolicyDescriptor.java).

---
In reply to: [200238905](https://github.com/Microsoft/pai/pull/812#discussion_r200238905) [](ancestors = 200238905)

Comment (fanyangCS) [reply frameworklauncher/doc/USERMANUAL.md]: I see. you introduce it in the comment of java code.

---
In reply to: [200240311](https://github.com/Microsoft/pai/pull/812#discussion_r200240311) [](ancestors = 200240311,200238905)

**Pull request merged**